{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "c1a93395_8f0d00fa",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 4
      },
      "lineNbr": 0,
      "author": {
        "id": 1373330
      },
      "writtenOn": "2022-04-20T21:07:04Z",
      "side": 1,
      "message": "I rerun the tests with your update and all look good. I left a comment about bypassing the presubmit issue. Otherwise, we are good to go. (no need to rush. ðŸ˜Š)",
      "revId": "e77850397c0e7db7ea806aa572ada02181de2f8e",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "6ceb19ea_cc4cae45",
        "filename": "dashboard/dashboard/pinpoint/models/change/patch.py",
        "patchSetId": 4
      },
      "lineNbr": 13,
      "author": {
        "id": 1373330
      },
      "writtenOn": "2022-04-20T21:07:04Z",
      "side": 1,
      "message": "using:\nimport six.moves.urllib.parse as urlparse\nand then we don\u0027t need to update line152-153. Then we can bypass the presubmit error.",
      "range": {
        "startLine": 13,
        "startChar": 7,
        "endLine": 13,
        "endChar": 29
      },
      "revId": "e77850397c0e7db7ea806aa572ada02181de2f8e",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}